tasks/configure-Debian.yml

```diff
@@ -23,6 +23,11 @@
   with_items: apache_mods_disabled
   notify: restart apache
 
+- name: Stat SSL certificates.
```

Comment:
Just to make this a little more clear, can you name it `name: Check whether certificates defined in vhosts exist.`?